devices/paulmann.js

```diff
@@ -108,4 +108,27 @@ module.exports = [
             'on', 'off', 'toggle', 'brightness_step_up', 'brightness_step_down', 'color_temperature_move', 'color_move', 'brightness_stop',
             'brightness_move_down', 'brightness_move_up', 'color_loop_set', 'enhanced_move_to_hue_and_saturation', 'scene_*'])],
     },
+    {
+        zigbeeModel: ['RGBCW_LIGHT'],
+        model: '501.27',
+        vendor: 'Paulmann',
+        description: 'Paulmann Light',
+        fromZigbee: [fz.color_colortemp, fz.on_off, fz.brightness, fz.level_config, fz.ignore_basic_report],
```

Comment:
The `fromZigbee`, `toZigbee`, `exposes`, `meta` and `configure` can all be replaced by `extend: extend.light_onoff_brightness_colortemp_color(),`

Make sure to also add the color temp range: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#31-retrieving-color-temperature-range-only-required-for-lights-which-support-color-temperature

Reply:
You're right, made the changes and it's still working.

Reply:
Thank you!